### PR TITLE
Docs OG-Image and Hamburger Color Fix

### PR DESCRIPTION
**Pull request by iamseeley**
Added custom Head component with og meta tags and fixed hamburger menu color in dark mode

### CODE REVIEW

File: src/styles/base.css [hunk 1]
```diff
@@ -135,3 +135,7 @@ dialog {
 p {
   text-wrap: pretty;
 }
+
+button:where(.astro-jif73yzw) {
```
Comment (pmillspaugh) [inline src/styles/base.css:139]: Hm, I think the classname `.astro-<id>` is autogenerated and won't be consistent, no?

Comment (pmillspaugh) [reply src/styles/base.css:139]: Or is it consistent? I see it works on the preview deployment

Comment (pmillspaugh) [inline src/styles/base.css:139]:
```suggestion
starlight-menu-button > button > svg {
```

Comment (pmillspaugh) [reply src/styles/base.css:139]: That'll work too^. Seems safer, yeah?

Comment (iamseeley) [reply src/styles/base.css:139]: Definitely, thanks!

File: src/styles/base.css [hunk 2]
```diff
@@ -135,3 +135,7 @@ dialog {
 p {
   text-wrap: pretty;
 }
+
+starlight-menu-button > button > svg {
```
Comment (pmillspaugh) [inline src/styles/base.css:139]: Ah sorry, we also need to select for dark mode
```suggestion
:root[data-theme="dark"] starlight-menu-button > button > svg {
```

**Pull request merged**